test/built-ins/Date/UTC/return-value.js

```diff
@@ -14,9 +14,11 @@ info: |
   7. If ms is supplied, let milli be ? ToNumber(ms); else let milli be 0.
   8. If y is not NaN and 0 ≤ ToInteger(y) ≤ 99, let yr be 1900+ToInteger(y);
      otherwise, let yr be y.
-  9. Return TimeClip(MakeDate(MakeDay(yr, m, dt), MakeTime(h, min, s, milli))). 
+  9. Return TimeClip(MakeDate(MakeDay(yr, m, dt), MakeTime(h, min, s, milli))).
 ---*/
 
+assert.sameValue(Date.UTC(1970), 0, '1970');
```

Comment:
We didn't have tests for the single arg cases before that now we could have some interesting cases.

Can we test some variation here? Instead of just asserting 1970, it would be great to include another year (2016, maybe?)